### PR TITLE
[zuul] Add functional-tests job

### PR DESCRIPTION
**Pull request by elfiesmelfie**
Currently, this job covers autoscaling and is subject to change

The new job is defined in https://github.com/infrawatch/feature-verification-tests/pull/82

### CODE REVIEW

**Comment by jlarriba** [comment]
This looks good to me, any reason of why it is in draft?

**Comment by elfiesmelfie** [comment]
> This looks good to me, any reason of why it is in draft?

Either because the previous PR I created was a draft, or because I wanted to make sure it worked before looking for reviews.

**Comment by elfiesmelfie** [comment]
recheck

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -78,3 +78,5 @@
         - telemetry-operator-multinode-autoscaling-tempest
         - telemetry-operator-multinode-default-telemetry
         - telemetry-operator-multinode-logging
+        - functional-tests-on-osp18:
+            voting: false
```
Comment (elfiesmelfie) [inline zuul.d/projects.yaml:82]:
```suggestion
            voting: false
            override-checkout: master
```

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -78,3 +78,6 @@
...
+            override-checkout: master
```
Comment (elfiesmelfie) [inline zuul.d/projects.yaml:83]:
```suggestion
            required-projects:
                - name: infrawatch/feature-verification-tests
                   override-checkout: master
```

File: zuul.d/projects.yaml [hunk 3]
```diff
@@ -78,3 +78,8 @@
         - telemetry-operator-multinode-autoscaling-tempest
         - telemetry-operator-multinode-default-telemetry
         - telemetry-operator-multinode-logging
+        - functional-tests-on-osp18:
+            voting: false
+            required-projects:
+                - name: infrawatch/feature-verification-tests
+                   override-checkout: master
```
Comment (elfiesmelfie) [inline zuul.d/projects.yaml:85]:
```suggestion
                  override-checkout: master
```

**Review by csibbitt — APPROVED** [review]

**Pull request merged**